ui/src/components/analyzer/analyzer-client.tsx

```diff
@@ -83,13 +98,44 @@ export function AnalyzerClient() {
     await analyzeProposal(selectedProposal);
   }
 
+  // Convert spaces to protocols for the header
+  const protocols: Protocol[] = spaces.map(space => ({
+    id: space.id,
+    name: space.name,
+    avatar: space.avatar,
+    verified: space.verified
+  }));
```

Comment:
This may render froever we need it only when spaces are updated
```suggestion
  const protocols: Protocol[] = useMemo(
    () => spaces.map(space => ({
      id: space.id,
      name: space.name,
      avatar: space.avatar,
      verified: space.verified
    })),
    [spaces]
  );
```